coretex/cli/modules/node.py

```diff
@@ -255,6 +255,23 @@ def promptRam(config: Dict[str, Any], ramLimit: int) -> int:
     return nodeRam
 
 
+def promptSwap(config: Dict[str, Any], swapLimit: int) -> int:
```

Comment:
`config` is only used to extract ram value, why not pass in just that, instead of whole config object?